README.md

```diff
@@ -269,13 +272,23 @@ Example of customization could be:
 spec:
   ...
   tower_image: myorg/my-custom-awx
+  tower_image_version: latest
   tower_image_pull_policy: Always
   tower_image_pull_secret: pull_secret_name
   tower_ee_images:
     - name: my-custom-awx-ee
       image: myorg/my-custom-awx-ee
 ```
 
+**Note**: The `tower_image` and `tower_image_version` are intended for local mirroring scenarios.
```

Comment:
I would add another sentence that says something like "Using a version of AWX other than the one bundled with the Operator is *not* supported"